openapi/components/schemas/Credentials/TaxJarCredential.yaml

```
@@ -47,5 +47,12 @@ properties:
     description: TaxJar API token.
     format: password
     example: 0ab15e3c1cc608b6f592fbddaddeb8f2
+  useFromAddress:
+    type: boolean
+    description: |-
+      Specifies if "from_" address fields must be sent to TaxJar.
```

suggestion: to use inline code highlighting
```suggestion
      Specifies if `from_` address fields must be sent to TaxJar.
```